CHANGELOG.md

```diff
@@ -9,11 +9,16 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Added
 
-- `rebar3_hank [Paulo Oliveira]
+- `rebar3_hank` [Paulo Oliveira]
```

Comment:
Oh, boy, this'll be all over the place 😄

Reply:
😄